ros/processor/event_producer.py

```diff
@@ -0,0 +1,71 @@
+import json
+from confluent_kafka import KafkaError
+from datetime import datetime, timezone
+from ros.lib.models import PerformanceProfile
+from ros.lib.config import NOTIFICATIONS_TOPIC, get_logger
+from ros.lib.utils import systems_ids_for_existing_profiles
+
+logger = get_logger(__name__)
+
+
+# Event for new suggestion
+def new_suggestion_event(host, platform_metadata, previous_state, current_state, producer):
+
+    org_id = host.get("org_id")
+    query = systems_ids_for_existing_profiles(org_id)
+    systems_with_suggestions = query.filter(PerformanceProfile.number_of_recommendations > 0).count()
+    request_id = platform_metadata.get('request_id')
```

Comment:
I would like to confirm that does engine message always include `platform_metadata` info?

Reply:
Yes,  it does